tests/unit/test_activation_config.py

```diff
@@ -36,7 +36,7 @@ def test_filtering_config_blank_when_activation_config_missing():
     assert any(elem.name == "pubsub_subscription" and elem.monitoring_filter == '' for elem in config)
     assert any(elem.name == "pubsub_snapshot" and elem.monitoring_filter == '' for elem in config)
 
-
-def test_filtering_missing_configs():
-    config = load_supported_services(context, [])
-    assert len(config) == 0
+# TODO BŁ NOMERGE! commenting out to have e2e test run
```

Comment:
so if it is ready to review this should be removed

Reply:
👍 I need to clear something for this in review when Maria comes back tomorrow

Reply:
Adapted after discussion with @mswiatkowska